Web muhammed

gktpmuhammed opened the pull request:
Landing page is ready. Please review and give feedback.

emiroztoprak commented:
All the changes seem to be working correctly. I think it is ready to merge.

4teko7 commented on Web/src/screens/bootstrap.js:
```diff
@@ -2,10 +2,11 @@
-import {Dropdown,ButtonGroup,Alert,Accordion,Button, DropdownButton,Spinner,Card, ProgressBar, Carousel} from 'react-bootstrap';
 import faker from 'faker/locale/tr';
 import { useSelector } from 'react-redux';
+import SideBar from 'components/navbar/SideBar';
```
You should remove SideBar from bootstrap screen. It is already in navigation.

4teko7 commented on Web/src/screens/bootstrap.js:
```diff
@@ -39,7 +38,9 @@ const Bootstrap = (props) => {
 </div>
-
+<div>
+<SideBar />  
```
This component is not necessary

4teko7 commented on Web/src/components/SignUpForm.js:
```diff
@@ -0,0 +1,65 @@
+import "../App.css"
+
+
+function Form() {
```
We are using React-Bootstrap. You should use components of React-Bootstrap. There are tons of components there. Currentlyi you are using normal html tags not components.

4teko7 commented on Web/src/components/LoginForm.js:
```diff
@@ -0,0 +1,43 @@
+import "../App.css"
+import { useHistory } from "react-router-dom";
+
+function Form() {
```
We are using React-Bootstrap. You should use components of React-Bootstrap. There are tons of components there. Currentlyi you are using normal html tags not components.

4teko7 commented on Web/src/App.css:
```diff
@@ -36,3 +36,160 @@
     transform: rotate(360deg);
   }
 }
+
```
Don't put lots of css in App.css
This file is valid for all pages. You should use component based css files. I mean, form.css login.css etc. can be created for some components. React-Bootstrap gives you lots of flexibility. You don't need to you lots of css.

4teko7 commented on Web/src/screens/landingPage.js:
```diff
@@ -0,0 +1,76 @@
+        <div class="latest-photos">
+            <div class="row"><div class="col-md-4"> 
+        <figure> <img class="img-fluid" src="https://i4.hurimg.com/i/hurriyet/75/1110x740/5b8e6d967152d827603dd434.jpg" alt=""/> 
+        </figure></div><div class="col-md-4"> <figure> <img class="img-fluid" src="https://i4.hurimg.com/i/hurriyet/75/1110x740/5b8e6d967152d827603dd434.jpg" alt=""/> 
```
With a map function you can create this lines.
let LINKS = [https://i4.hurimg.com/i/hurriyet/75/1110x740/5b8e6d967152d827603dd434.jpg, https://i4.hurimg.com/i/hurriyet/75/1110x740/5b8e6d967152d827603dd434.jpg,]
let FIGURES = []
LINKS.map(res => {
  FIGURES.push(
    <figure> <img class="img-fluid" src= `${res}` alt=""/> 
  )
})

{FIGURES}

gktpmuhammed commented:
I will close this pull request and open a new one. I have corrected some files based on the feedback from Bilal. I have added new functionalities aslo.
